README.md

```diff
@@ -1,2 +1,36 @@
 # nf-agora
-Nextflow workflow for agora-data-tools data processing
+
+## Intro
+
+This repository contains a Nextflow Pipeline that wraps agora-data-tools and therefore the Agora ETL process. It is deployed on Sage's Nextflow Tower Instance, making it simple to trigger and monitor runs. 
+
+## Running the pipeline on Nextflow Tower
+
+### Getting Oriented in Nextflow Tower
+
+1. Navigate to https://tower.sagebionetworks.org/login.
+2. Click “Sign in with Google” and enter your Sage credentials.
+3. In the top left corner, you should see “Sage-Bionetworks/agora-project”. This indicates that you are in the workspace for Agora in our Nextflow Tower instance.
+4. If you are not in the correct workspace, click on the drop-down and select “agora-project”.
+
+### Launching the Pipeline for Testing
+
+1. In the “Launchpad” tab, click on the tile labeled “nf-agora”.
+2. At the top, give your Workflow run a name if you want. If you leave it blank, a randomly generated name will be given to it.
+3. Under "Config profile", add `test`.
```

Comment:
@BWMac / @BrunoGrandePhD : Thoughts on adding a "prod" profile for convenience? Tagged you bruno because I'm not sure if "prod" is a standard we would want to adopt.

Reply:
If the parameters never change, then I think the use of a `prod` profile makes sense. I can't think of any downsides at the moment. It's not a convention within nf-core, but that's usually because you have to configure nf-core workflows for your data. This is—by design—not a general-purpose workflow.